docs/source/guides/configuration.rst

```diff
@@ -89,6 +93,14 @@ In Iroha logging can be adjusted as granularly as you want. Each component has i
   }
 
 Every part of this config section is optional.
-- ``level`` sets the verbosity. Available valies are (in decreasing verbosity order): trace, debug, info, warning, error, critical.
-- ``patterns`` controls the formatting of each log string for different verbosity levels. Each value overrides the less verbose levels too. So in the example above, the "don't panic" pattern also applies to info and warning levels, and the trace level pattern is the only one that is not initialized in the config (it will be set to default hardcoded value).
-- ``children`` describes the overrides of child nodes. The keys are the names of the components, and the values have the same syntax and semantics as the root log configuration.
+
+- ``level`` sets the verbosity. Available valies are (in decreasing verbosity
+  order): trace, debug, info, warning, error, critical.
+- ``patterns`` controls the formatting of each log string for different
+  verbosity levels. Each value overrides the less verbose levels too. So in the
```

Comment:
Please put each sentence on its own line.

Reply:
Each value overrides the less verbose levels too.
Probably can be rephrased. It looks like "values" and "levels" are the objects of the same kind. Not clear.